.travis.yml

```diff
@@ -48,5 +58,17 @@ script:
 
   - make pylint
   - make pycodestyle
-
+  
+  # build and deploy website
+  - openssl aes-256-cbc -K $encrypted_9bd3898936c8_key -iv $encrypted_9bd3898936c8_iv -in deploy-key.enc -out deploy-key -d
```

Comment:
Same with this -- we could move the keys to private Travis variables to avoid exposing them here

Reply:
The key is actually encrypted, there is public part hidden in github repository settings, so this should be ok in principle. It can be moved to travis but it is probably not necessary.

Reply:
If I get it correctly, which I cannot guarantee, it seems like travis gets the key from github as these are just references to variables in travis environment. So quite likely this is ok.